lib/plurimath/math/function/unary_function.rb

```diff
@@ -147,15 +165,39 @@ def latex_value
 
         def omml_value(display_style)
           if parameter_one.is_a?(Array)
-            return parameter_one&.compact&.map { |obj| obj.insert_t_tag(display_style) }
+            return parameter_one&.compact&.map { |object| formula_to_nodes(object, display_style) }
           end
 
-          Array(parameter_one&.insert_t_tag(display_style))
+          Array(formula_to_nodes(parameter_one, display_style))
+        end
+
+        def formula_to_nodes(object, display_style)
+          object&.insert_t_tag(display_style)
         end
 
         def latex_paren
           Latex::Constants::LEFT_RIGHT_PARENTHESIS.invert[parameter_one] || '.'
         end
+
+        def exist?
+          parameter_one && !(parameter_one.is_a?(Array) ? parameter_one.empty? : parameter_one.nil?)
```

Comment:
Because of first condition second `parameter_one.nil?` will always be false. Suggest changing it to something like

```suggestion
          return !parameter_one.empty? if parameter_one.is_a?(Array)
          
          !parameter_one.nil?
```